packages/truffle-core/lib/debug/printer.js

```diff
@@ -296,7 +296,7 @@ class DebugPrinter {
     try {
       let result = safeEval(expr, context);
       result = DebugUtils.cleanConstructors(result); //HACK
-      const formatted = DebugUtils.formatValue(result, indent);
+      const formatted = DebugUtils.formatValue(result, indent, true);
```

Comment:
How do you feel about a breaking change to debug-utils to use a signature with options:
`formatValue (value, { indent, nativized })`?

`indent` is fine because it's almost certainly always going to be a variable, but seeing `true` means one has to go look up the function definition to see what it means.

Reply:
I mean, I could?  I'm not sure this is substantial enough to be worth it; this is all pretty internal stuff...

Reply:
Up to you!

Reply:
Cool, think I'll leave this as-is then.